src/components/Play/Play.js

```diff
@@ -68,44 +72,75 @@ const Play = () => {
   };
 
   useEffect(() => {
-    fetchKit("Andromeda%20Strain").then((data) => setKit(data));
+    fetchKit("Andromeda%20Strain")
+      .then((data) => {
+        setKit(data);
+        setTimeout(() => {
+          setLoading(false);
+        }, 2000);
+      })
+      .catch((e) => {
+        setLoading(false);
+        setErrorMessage("Sorry, We were unable to get that. Please try again!");
+      });
   }, []);
 
   useEffect(() => {
-    if (isFirstLoad.current) {
-      isFirstLoad.current = false;
-    } else {
+    // if (isFirstLoad.current) {
```

Comment:
⛏️ I don't know if you still need this code, but if you don't perhaps consider just deleting it from your repo and restoring if you need it later.